File: scripts/fix-blog.js
const fs = require('fs');
let code = fs.readFileSync('src/lib/templates/SiteBlogIndex.tsx', 'utf-8');

code = code.replace(
  'export default function SiteBlogIndex(',
  'export default async function SiteBlogIndex('
);

fs.writeFileSync('src/lib/templates/SiteBlogIndex.tsx', code);

File: scripts/fix-blog.py
with open("src/lib/templates/SiteBlogIndex.tsx", "r", encoding="utf-8") as f:
    text = f.read()

text = text.replace("export default function SiteBlogIndex(", "export default async function SiteBlogIndex(")

with open("src/lib/templates/SiteBlogIndex.tsx", "w", encoding="utf-8") as f:
    f.write(text)
print("Fixed blog")
